### DISCUSSION TITLE
Instructions for setting up YT-Spammer-Purge on Docker

### DISCUSSION THREAD

**Discussion by Techboy** [ThioJoe/YT-Spammer-Purge#905] [Q&A] [answered]
https://github.com/ThioJoe/YT-Spammer-Purge/blob/main/README.md#docker-instructions has high-level instructions for setting up YT-Spammer-Purge on Docker, but I'm relatively new to Docker so would like some more instruction :-)

I have set it up on my Windows PC and altered the SpamPurgeConfig.ini file so it can fully automated in Windows, but I want to set it up on Docker within my Synology NAS.

I have set up a Docker container, copied my files **client_secrets.json** and **SpamPurgeConfig.ini** into the directory **/usr/src/app** and run **YTSpammerPurge.py**. It displays the message:

> Loading YT Spammer Purge @ 2.16.9...
> Please login using the browser window that opened just now.
> Waiting for authorization. See message above.

It doesn't open a browser window on my PC.

The instructions say '**you must run the YTSpammerPurge.py script at least once with your client_secrets.json file**' - how do I do this?

**Comment by Kurdakov** [comment, marked as answer]
you should also copy tokens.pickle file which will be generated after your first login on windows, it is explained on main page:

> Before running docker-compose you must run the YTSpammerPurge.py script at least once with your client_secrets.json file to confirm OAuth credentials and generate the config/token files.

**Reply by Techboy** [reply]
Ah, I missed that file! 😊

I have copied it over, run **./YTSpammerPurge.py** and it works!

Now I just need to schedule it to run every hour in my Synology.

Thanks 👍
